packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx

```diff
@@ -50,31 +50,30 @@ export const CollectibleList = memo<CollectibleListProps>(({ selectedNetwork })
     const [page, setPage] = useState(0)
     const navigate = useNavigate()
     const account = useAccount()
-    const { Asset } = useWeb3PluginState()
-    const network = useNetworkDescriptor()
+    const chainId = useChainId()
     const [loadingSize, setLoadingSize] = useState(0)
-    const [renderData, setRenderData] = useState<Web3Plugin.NonFungibleToken[]>([])
+    const customCollectibles = useCustomNonFungibleAssets(
+        account,
+        chainId,
+        selectedNetwork?.ID === `${PluginId.EVM}_ethereum`,
+    )
 
     const {
-        value = { data: EMPTY_LIST, hasNextPage: false },
-        error: collectiblesError,
-        loading: isQuerying,
+        data: _collectibles,
+        state: loadingCollectibleDone,
         retry,
-    } = useAsyncRetry(
-        async () =>
-            Asset?.getNonFungibleAssets?.(account, { page: page, size: 20 }, undefined, selectedNetwork || undefined),
```

Comment:
#why_remove_getNonFungibleAssets

This is weird, to using this `Asset.getNonFungibleAssets`, we have to wrap it in a useAsyncRetry hook.